client/src/utils/data-formats/formats/bars.ts

```diff
@@ -59,25 +58,11 @@ export function bars(
   const valueField = args.valueField || 'count';
   const barsField = args.barsField || null;
   const seriesField = args.seriesField || null;
-  const valueMaxLength = args.valueMaxLength && parseInt(args.valueMaxLength, 10) || 13;
   const threshold = args.threshold || 0;
   const othersValue = args.othersValue || 'Others';
 
   let values: any[] = state.values || [];
 
-  // Concating values with '...'
-  if (values && values.length && valueMaxLength && (seriesField || barsField)) {
```

Comment:
wait... do we still need this block if 'valueMaxLength' is removed? The css trims the long legend labels in the chart.